### ISSUE TITLE
_Dsgnsystm: Mobile Menu cannot be toggled via keyboard navigation

### ISSUE THREAD

**Issue by thomasguillot** [Automattic/themes#1069]
## Steps to replicate

Screencast: https://cld.wthms.co/Qw6WSL

Note: We should consider removing the dropdown-icon from the button (maybe introduce a hamburger/close icon?)

**Comment by thomasguillot** [comment]
_Side note: not quite sure why the button in the cover blocks cannot be focused_
